doc/release_notes.rst

```diff
@@ -22,6 +22,11 @@ Upcoming Release
 
 * When solving ``n.lopf(pyomo=False, store_basis=True, solver_name="cplex")`` an error raised by trying to store a non-existing basis is caught.
 
+* Integration of inverter control strategies and on load tap changer transformer for non linear power flow:
+   Inverter control strategies such as: fix power factor, reactive power as a function of voltage Q(U),
+   Active power as a function of voltage P(U) and  power factor (PF) as a function of active power PF(P) can be
+   can be implemented on Load, Generator, Store and StorageUnit components as new attributes for these components
+   and also application of On-Load tap changer transformer as attribute of transformer would be interesting to be implemented. 
```

Comment:
```suggestion
* Integration of inverter control strategies and on-load tap changer (OLTC) transformer for nonlinear power flow:
   Inverter control strategies such as: fixed power factor, reactive power as a function of voltage Q(V),
   active power as a function of voltage P(V) and power factor (PF) as a function of active power PF(P) can be
   can be implemented on Load, Generator, Store and StorageUnit components by specifying new attributes.
```

Removed last bit on OLTCs because release note should only include new features.

PF abbreviation for power factors leads to some ambiguity with the power flow function `n.pf()`. Maybe we can use another or no abbreviation?

I think, we have used V for voltage so far, not U.